content/en/docs/reference/command-line-tools-reference/feature-gates/multi-cidr-service-allocator.md

```diff
@@ -9,5 +9,9 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.27"
+    toVersion: "1.30"
+  - stage: beta
+    defaultValue: false
```

Comment:
This feature is promoted to Beta and its default remains "false"?
Why the promotion then?

Reply:
Since https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default the betas APIs are off by default, this present a challenge for new features that depend on new APIS.

It depends on each feature you can choose to be feature-gate and API beta both to be on or off by default, but not independently, in this case we apply  https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default and we switch the API off , hence the feature has to be off or it will fail to run.

Reply:
Hello @aojea :wave:! v1.31 Doc Lead here. Just checking in as we approach Docs Freeze on Tuesday July 30th 18:00 PDT. This documentation appears to still be under review. To meet the Docs Freeze, this PR must have a technical review as well as lgtm and approve labels applied, without any unaddressed comments or concerns from SIG Docs. The status of this enhancement is marked as at risk for docs freeze. Thank you!

Reply:
this is just updating a feature gate, so no much technical to review

@danwinship @thockin  can you PTAL and LGTM if you are ook with it?

Reply:
@aojea Thanks for the explanation. So the "root cause" is that beta APIs are not enabled by default, just as alpha APIs... sigh.

/approve

Reply:
nope, sorry, is not about enabling, is about being served byt the apiserver, alpha and beta APIs are not served by default , you need to enable them in the apiserver with the `-runtime-config` flag.